recipes/pandoc-panflute/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "panflute" %}
+{% set version = "2.0.5" %}
+{% set sha256 = "ff6b65624442ed4a331133326ca001f4043a6552e6d344e87dc085943dc355eb" %}
+
+
+package:
+  name: pandoc-{{ name|lower }}
```

Comment:
why add a `pandoc-` prefix?

Reply:
Thanks. Because this one has pandoc as the dependencies. There exist another conda forge packages without the prefix: panflute. That one does not depends on pandoc.

For some people they might have pandoc installed in various ways such that they might not want the pandoc executable reinstalled in the environment. So here are the choices.

The idea is to have a whole set of pandoc-* conda forge packages that depends on pandoc. As panflute implements the pandoc AST which is sensitive to the pandoc API version, having a proper dependency management including the pandoc version is going to help people avoid a lot of unexpected bugs.

The next one to come will be pandoc-pantable after this, where I already have pantable in PyPI.

Let me know if the naming scheme make sense. Thanks!

Reply:
Doesn't make sense to me. If somebody wants pandoc, why can't they just do `conda install pandoc pantable`?

Reply:
because of the dependencies. panflute 2 only supports pandoc>=2.11.0.4. panflute 1 supports pandoc < 2.10.

Reply:
Essentially we are using conda to properly managing between pandoc (a binary built from Haskell source) and Python libraries that uses pandoc, which all of them currently call pandoc as a cli program, not at a dynamic library level. It has been a nightmare when these 2 mismatches, especially after pandoc 2.10 drops a major change in its AST that created chaos (no body's fault, we just don't have the infrastructure to do that.) I'm experimenting with using conda to fill that gap.

Reply:
you can have `run_constrained` in `panflute` to restrict the versions of `pandoc` installed into the same environment.

Reply:
@ickc I've not read through this PR properly but it looks like your latest comment would be satisfied by this suggestion?

If the pre-existing package binaries on the channel are a problem a [repodata patch](https://github.com/conda-forge/conda-forge-repodata-patches-feedstock) could be used to add `run_constrained` to them retroactively.